_inc/client/at-a-glance/akismet.jsx

```diff
@@ -158,16 +138,8 @@ DashAkismet.propTypes = {
 export default connect(
 	( state ) => {
 		return {
-			getAkismetData: () => _getAkismetData( state ),
-			isModuleActivated: ( module_name ) => _isModuleActivated( state, module_name ),
-			getSitePlan: () => getSitePlan( state )
+			getAkismetData: getAkismetData( state ),
+			getSitePlan: getSitePlan( state )
```

Comment:
These two prop names are confusing now that they're no longer functions. `akimestData` and `getSitePlan` would be clearer